index.html

```diff
@@ -18,8 +18,8 @@
 </head>
 
 <body>
-   <h1 style="font-size:50px"><span style="color:black">// M A C D E C O U R C Y</span></h1>
-   <h1>software engineer</h1>
+   <h1 style="font-size:85px"><span style="color:black">// M A C D E C O U R C Y</span></h1>
```

Comment:
I would do: `<h1 classname='.header-name'>// M A C D E C O U R C Y</h1>`
then create css classes. Every style thing you want to happen in all classes that begin with .header go in that class and get inherited by .header-name unless you write the property out as something different (which overrides it):
`.header {...}
.header-name {...}
.header-jobTitle {...}`